src/strauss/channels.py

```diff
@@ -53,9 +56,39 @@ def __init__(self, phi, mic_type="directional", label="C", channel=1):
             self.antenna = lambda a, b=0.5*np.pi: a**0.
         elif mic_type == "mute":
             self.antenna = lambda a, b=0.5*np.pi: a*0.
+        elif mic_type == "ambisonic":
+            self.antenna = self._ambisonic_antenna(acn=phi)
         else:
-            raise Exception(f"Mic type \"{mic_type}\" unknown.")        
+            raise Exception(f"Mic type \"{mic_type}\" unknown.")
         
+    def _ambisonic_antenna(self, acn):        
+        # get order and degree of spherical harmonic from ACN using ambiX standard
+        # (see iem.kug.ac.at/fileadmin/media/iem/projects/2011/
+        # ambisonics11_nachbar_zotter_sontacchi_deleflie.pdf, footnote 5)
```

Comment:
Broken link

Reply:
here's a working link: https://iaem.at/ambisonics/proceedings-of-the-ambisonics-symposium-2011/ambix-a-suggested-ambisonics-format/view

will find something more permanent to go into the code comments